pnr/nextpnr/fpga_interchange/build.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -e
+set -x
+
+export PYTHON_EXECUTABLE=`which python3`
+
+# Install rapidyaml
+python3 -m pip install git+https://github.com/diegoferigo/cmake-build-extension.git
+python3 -m pip install https://files.pythonhosted.org/packages/55/a5/66e07773163aa004812f00e9f3529531f9d5715fe290ce4b085f6e7856c0/rapidyaml-0.1.0.post60.tar.gz
```

Comment:
`cmake-build-extension` is needed when installing from `tar.gz`. I had to use the precise URL for the `rapidyaml` package because simple `python3 -m pip install rapidyaml` doesn't work when called from `build.sh`

Reply:
The version of rapidyaml used in the CI script for nextpnr should be the one used in the python-fpga-interchange [requirements](https://github.com/SymbiFlow/python-fpga-interchange/blob/master/requirements.txt) file. Doing a pip install -r requirements.txt in the python-fpga-interchange should be enough.

Reply:
But ultimately, I believe that rapidyaml is not strictly necessary to build the device databases

Reply:
Hmm, and why using the requirements file from python-fpga-interchange which contains the following cannot be adopted in this case?
```
git+https://github.com/litghost/rapidyaml.git@fixup_python_packaging#egg=rapidyaml
```

Reply:
The `rapidyaml` package:
```
git+https://github.com/litghost/rapidyaml.git@fixup_python_packaging#egg=rapidyaml
```
is just a temporary solution for missing PyPI package and I think this is an outdated requirement since the `rapidyaml` is available on PyPI, thus I used the URL to PyPI package. Additionally installing it explicitly in `build.sh` will also need `cmake-build-extension` package. I'm not convinced we should install everything from `requirements.txt` either, because we don't need some packages from there.

In general we should stick to conda packages where it's possible (this is a suggested way by anaconda) and because the `rapidyaml` is not available in anaconda I think it would be better to switch to `pyyaml`. In the end it shouldn't matter whether we use `pyyaml` or `rapidyaml` (probably the difference would be in build time?)